homeassistant/components/template/strings.json

```diff
@@ -527,6 +527,10 @@
     "deprecated_battery_level": {
       "description": "The template vacuum options `battery_level` and `battery_level_template` are being removed in 2026.8.\n\nPlease remove the `battery_level` or `battery_level_template` option from the YAML configuration for {entity_id} ({entity_name}).",
       "title": "Deprecated battery level option in {entity_name}"
+    },
+    "deprecated_legacy_templates": {
+      "description": "The legacy `platform: template` syntax for `{domain}` is being removed in Home Assitant Core 2026.6.\n\nPlease migrate to the modern template syntax.\n\n```\n{config}\n```",
```

Comment:
Any chance we can add something as identifier in this text. Potentially users might get loads of these issues if they have many legacy templates.